basic-tests.ts

```diff
@@ -308,7 +308,10 @@ export function hashToken(token: string) {
   return createHash("sha256").update(`${token}anything`).digest("hex")
 }
 
-export { randomUUID }
+function randomUUID() {
+  // Use `randomUUID` if available. (Node 15.6++)
+  return cryptoRandomUUID?.() ?? randomBytes(16).toString("hex")
+}
```

Comment:
Could you please revert these changes? You should be able to get around this if you install Node 16. We don't wish to maintain code for multiple Node versions for the sake of testing.